server/repository/src/db_diesel/program_event_row.rs

```diff
@@ -28,14 +28,18 @@ pub struct ProgramEventRow {
     /// Time of the event. An event can be "invalidated" by a later event of same type.
     pub datetime: NaiveDateTime,
     /// Time when the event becomes active.
-    /// Must be <= datetime.
+    /// Must be >= datetime.
     /// An event could never become active if there is another event e2 if
     /// e.active_datetime > e2.datetime && e.datetime < e2.datetime
     /// i.e. e2 superseded the event.
+    /// The active_start_datetime a constant property of the event, e.g. it will not be updated
+    /// when other events are insert.
     pub active_start_datetime: NaiveDateTime,
     /// Keeps track when the event becomes superseded by a later event.
     /// Its possible that active_end_datetime < active_start_datetime in which case the event never
-    /// became active
+    /// became active.
+    /// Other than the active_start_datetime the active_end_datetime might get updated when other
+    /// events are inserted, i.e. it depends on other events in the system.
```

Comment:
Just updating the docs with some questions I had when looking at the code again...